samples/property/provider/src/main.rs

```diff
@@ -60,31 +66,41 @@ async fn register_ambient_air_temperature(
     Ok(())
 }
 
+fn create_property(ambient_air_temperature: i32) -> String {
+    let metadata = Metadata { model: sdv::hvac::ambient_air_temperature::ID.to_string() };
+
+    let property: Property = Property { ambient_air_temperature, metadata };
+
+    serde_json::to_string(&property).unwrap()
+}
+
 /// Start the ambient air temperature data stream.
 ///
 /// # Arguments
 /// `id_to_subscribers_map` - The id to subscribers map.
 fn start_ambient_air_temperature_data_stream(subscription_map: Arc<Mutex<SubscriptionMap>>) {
     debug!("Starting the Provider's ambient air temperature data stream.");
     tokio::spawn(async move {
-        let mut temperature: u32 = 75;
+        let mut temperature: i32 = 75;
```

Comment:
Freyja uses f32 for temperature values, mostly to support better unit conversions. We should consider adopting this data type across all projects for temperature

Reply:
Let discuss this when you get back from vacation.